nxviz/plots.py

```diff
@@ -68,10 +69,12 @@ class BasePlot(object):
     :type edgeprops: `dict`
     """  # noqa
     def __init__(self, graph, node_order=None, node_size=None,
-                 node_grouping=None, node_color=None, node_labels=None,
-                 edge_width=None, edge_color=None, data_types=None,
-                 nodeprops=None, edgeprops=None, node_label_color=False,
-                 group_label_position=None, group_label_color=False, **kwargs):
+                 node_grouping=None, group_order="alphabetically",
```

Comment:
In the diff screen, it appears that `node_grouping` and `group_order` aren't documented in the docstring above (should be inserted after line 54 and before line 56). Could you update the docstring please?

Reply:
Ah, got it! Yes, please add a docstring for group_label_position then, so that others know that this option is available.